### ISSUE TITLE
Torrents EU failed configuration

### ISSUE THREAD

**Issue by insanebits** [Jackett/Jackett#517]
TorrentsEU tracker seems to be broken, it might be due to changed domain: cinemaz.to

Here are relevant logs, as you can see it's POST'ing to old url and getting 301 redirect
```
Sep 18 16:10:40 sonar mono[14630]: 09-18 16:10:40 Debug UnixLibCurlWebClient:GetString(Url:https://eutorrents.to/auth/login)
Sep 18 16:10:40 sonar mono[14630]: 09-18 16:10:40 Debug UnixLibCurlWebClient: Posting _token=&email_username=XXXXXXX&password=XXXXXXX&remember=1
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Debug UnixLibCurlWebClient:GetString Returning MovedPermanently => <html>
Sep 18 16:10:41 sonar mono[14630]: <head><title>301 Moved Permanently</title></head>
Sep 18 16:10:41 sonar mono[14630]: <body bgcolor="white">
Sep 18 16:10:41 sonar mono[14630]: <center><h1>301 Moved Permanently</h1></center>
Sep 18 16:10:41 sonar mono[14630]: <hr><center>nginx</center>
Sep 18 16:10:41 sonar mono[14630]: </body>
Sep 18 16:10:41 sonar mono[14630]: </html>
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Debug UnixLibCurlWebClient:GetBytes(Url:https://cinemaz.to/auth/login)
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Debug UnixLibCurlWebClient:GetBytes Returning OK => 5070 bytes
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info ReflectedHttpActionDescriptor ExecuteAsync Action returned System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]'
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info ApiControllerActionInvoker InvokeActionAsync
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Action: ApiControllerActionInvoker InvokeActionAsync
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()'
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()'
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info JackettAPINoCacheAttribute OnActionExecutedAsync
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info AdminController ExecuteAsync
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Controllers: AdminController ExecuteAsync
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info   Content-type='application/json; charset=utf-8', content-length=339
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Request:   Content-type='application/json; charset=utf-8', content-length=339
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info AdminController Dispose
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Controllers: AdminController Dispose
Sep 18 16:10:41 sonar mono[14630]: 09-18 16:10:41 Info AdminController Dispose
Sep 18 16:10:41 sonar mono[14630]: System.Web.Http.Controllers: AdminController Dispose
```

**Comment by kaso17** [comment]
update to the latest version

**Comment by insanebits** [comment]
I am running latest "Jackett Version 0.7.380.0"

**Comment by kaso17** [comment]
I'm sorry. It's fixed with #498 but there's no build for it yet. You'll have to build it yourself if you need it.

**Comment by kaso17** [comment]
With the latest release v0.7.398 this should be fixed now (you have to readd the cinemaZ tracker)
